### PR TITLE
[RELAY][OP]  Maketuple to be resolved when containing incompleteType

### PR DESCRIPTION
**Pull request by tqchen**
THanks @siju-samuel for reporting this

### CODE REVIEW

**Comment by tqchen** [comment]
@siju-samuel @junrushao1994  please review

File: tests/python/relay/test_op_level1.py [hunk 1]
```diff
@@ -96,6 +97,9 @@ def test_concatenate_infer_type():
     assert zz.checked_type == relay.TensorType((n, t + t, 100))
 
 
+
```
Comment (fletchersm) [inline tests/python/relay/test_op_level1.py]: Remove blank lines here?

Comment (junrushao1994) [inline tests/python/relay/test_op_level1.py]: Remove extra blank lines?

**Comment by tqchen** [comment]
Thanks @siju-samuel @junrushao1994 , this is now merged

**Pull request merged**